crates/amaru-sim/src/bin/amaru-sim/bytes.rs

```diff
@@ -0,0 +1,53 @@
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
```

Comment:
```suggestion

// Copyright 2025 PRAGMA
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

use serde::{Deserialize, Deserializer, Serialize, Serializer};
```

Reply:
I really need to add a template to my Emacs setting :)